### ISSUE TITLE
Cant Compile

### ISSUE THREAD

**Issue by shaddow501** [cmnybo/chip_stuff#1]
Dear Sir
When I try to compile on the GETCHIP board (I didnt try anywhere else) I get this error:

What overlay do you want to compile:
mmc.dts 
Compiling mmc.dts
mmc.dts:31:43: fatal error: dt-bindings/pinctrl/sun4i-a10.h: No such file or directory
 #include <dt-bindings/pinctrl/sun4i-a10.h>
                                           ^
compilation terminated.
Error: mmc.dts:29.9-10 syntax error
FATAL ERROR: Unable to parse input tree
There was an error during compilation

What am I missing, I didnt find anywhere what else should I install in order to compile it successfully.
Can you post some instructions?

**Comment by cmnybo** [comment]
You are getting the error because your are missing the `sun4i-a10.h` file.
You need to download the source for your Linux kernel and copy the `arch/arm/boot/dts/include/dt-bindings` directory to `/usr/include`.

**Comment by shaddow501** [comment]
in https://github.com/NextThingCo/CHIP-linux/tree/debian/4.4.13-ntc-mlc/arch/arm/boot/dts/include
I dont see any files.

**Comment by cmnybo** [comment]
It looks like the files are available in [include/dt-bindings](https://github.com/NextThingCo/CHIP-linux/tree/debian/4.4.13-ntc-mlc/include/dt-bindings) on that kernel.

**Comment by shaddow501** [comment]
Yes that is correct just copied them and managed to compile without any issues
Thanks.
Also it is better to put the Devtree.sh in the device_trees folder or else it cant find them..
